Dockerfile

```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM golang:1.24.2@sha256:991aa6a6e4431f2f01e869a812934bd60fbc87fb939e4a1ea54b8494ab9d2fc6 AS builder
+FROM golang:1.24.2@sha256:fb224f950b0dfb889f8f1122bf3dfc21976735ccf983b73a1e6e014215a272f5 AS builder
```

Comment:
Can't tell what this hash change means, is this a newer or older version? It's not the latest dockerhub version of 1.24.2.

Reply:
It's been updated again, this was the hash yesterday. :| Will update, `crane digest golang:1.24.2` now returns `sha256:1ecc479bc712a6bdb56df3e346e33edcc141f469f82840bab9f4bc2bc41bf91d`